config/crds/cluster.k8s.io_machines.yaml

```diff
@@ -6,29 +6,13 @@ metadata:
   creationTimestamp: null
   name: machines.cluster.k8s.io
 spec:
-  additionalPrinterColumns:
-  - JSONPath: .spec.providerID
-    description: Provider ID
-    name: ProviderID
-    type: string
-  - JSONPath: .status.phase
-    description: Machine status such as Terminating/Pending/Running/Failed etc
-    name: Phase
-    type: string
-  - JSONPath: .status.nodeRef.name
-    description: Node name associated with this machine
-    name: NodeName
-    priority: 1
-    type: string
   group: cluster.k8s.io
   names:
     kind: Machine
     plural: machines
     shortNames:
     - ma
   scope: Namespaced
-  subresources:
-    status: {}
```

Comment:
What happened to the status subresource?

Reply:
Same as above